lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextManager.cs

```diff
@@ -33,18 +34,24 @@ namespace Org.Apache.REEF.Common.Runtime.Evaluator.Context
 {
     public sealed class ContextManager : IDisposable
     {
-        private static readonly Logger LOGGER = Logger.GetLogger(typeof(ContextManager));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(ContextManager));
```

Comment:
This blows up this particular change a lot. Hence, it might be a good idea to move the rename of *all* `LOGGER` to `Logger` into a separate JIRA? That way, we can separate pure style changes from functionality changes in our commits.